code/modules/antagonists/spy/spy_bounty.dm

```diff
@@ -639,7 +639,7 @@
 	theft_time = 10 SECONDS
 	black_market_prob = 0
 	/// What typepath of bot we want to steal.
-	var/mob/living/simple_animal/bot/bot_type
+	var/mob/living/bot_type
```

Comment:
oh my god

Reply:
closest ancestor of `/mob/living/basic/bot` and `/mob/living/simple_animal/bot` is `/mob/living` idk what you want from me